### PR TITLE
Remove 10 panels; Rename panels for symmetry.

### PR DESCRIPTION
**Pull request by jvyduna**

### CODE REVIEW

**Comment by jvyduna** [comment]
Please let me know if the viewport is messed up for you (off center; [colors bleed](https://monosnap.com/file/oALixBdUZoTJ1najHiFfpMyGc9nifi))

File: LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java [hunk 1]
```diff
@@ -128,12 +128,13 @@ public void onDraw(UI ui, PGraphics pg) {
       if (this.panelLabelsVisible.getValueb()) {
         pg.pushMatrix();
         LXVector centroid = p.centroid;
-        pg.translate(centroid.x, centroid.y, centroid.z);
+        pg.translate(centroid.x * 1.12f, centroid.y, centroid.z * 1.02f);
```
Comment (raldi) [inline LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java]: Can you turn these two constants into variables with meaningful names?

File: LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java [hunk 2]
```diff
@@ -128,12 +128,13 @@ public void onDraw(UI ui, PGraphics pg) {
...
         //pg.rotateY((float) (-Math.PI / 2.0));  // Face port (non-show) side
-        pg.rotateY((float) (Math.PI / 2.0));  // Face starboard (show) side
+        pg.rotateY((float) (Math.atan2(centroid.x, centroid.z/5) + Math.PI));  // Face out
```
Comment (raldi) [inline LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java:136]: Ooooh

Edit: In addition to this new line, can we keep the old line 133, commented out? And maybe explain what the "centroid.z/5" is about?

**Review by raldi — APPROVED** [review]
Approved with one nitpick. Oh, and also, just want to check that you're submitting "te - Jeff.lxp" on purpose. I figured we would just have one shared lxp file representing sane defaults, but if you want to check your personal one in as well, that's fine with me.

File: LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java (hunk 1 above)
Comment (jvyduna) [reply LXStudio-IDE/src/main/java/titanicsend/app/TEVirtualOverlays.java]: Comments added, thanks.

File: LXStudio-IDE/Projects/te.lxp [hunk 3]
```diff
@@ -427,22 +427,22 @@
...
-          "phi": 0.0,
-          "x": 4.5,
-          "y": 0.0,
-          "z": 0.0
+          "radius": 1.6700490355594486E7,
+          "theta": -1.6295745824463665,
+          "phi": 0.2857836023904383,
+          "x": 776130.9353027344,
+          "y": 2626079.4140625,
+          "z": 54030.869079589844
```
Comment (jvyduna) [inline LXStudio-IDE/Projects/te.lxp:435]: I believe these lines help the te.lxp project have our entire model in view by default (so new people do not need to scale out to many-micron view)

Comment (raldi) [reply LXStudio-IDE/Projects/te.lxp:435]: 📷👍🏼

**Review by raldi — APPROVED** [review]

**Comment by jvyduna** [comment]
Confirming I wanted to check in my personal one; felt it'd be useful for exchanging project-specific ideas. I also deleted the old `Playground - Jeff.lxp` to keep it tidier.

**Pull request merged**